### ISSUE TITLE
How to use multiple gpus?

### ISSUE THREAD

**Issue by mzshuo** [shilrley6/Faster-R-CNN-with-model-pretrained-on-Visual-Genome#12]
I set the argument '--mGPUs' but the model still only used one gpu.
And I checked the 'generate_tsv.py' code and found variable '--mGPUs' is not used except setting.
Does anyone face the same problem?

**Comment by Alchemistyui** [comment]
I want to know too... Did you solve it?

**Comment by mzshuo** [comment]
@Alchemistyui 

Yes. I split the dataset to 4 folds and feed them to the model separately using 4 different gpu set manually.

Specifically, add this line at the head of python file.
`import os`
`os.environ["CUDA_VISIBLE_DEVICES"] = args.gpu`

**Comment by Alchemistyui** [comment]
@shuorr Thank you very much!
